src/Features/Core/Portable/CodeStyle/AbstractCodeStyleDiagnosticAnalyzer.cs

```diff
@@ -109,7 +85,7 @@ protected DiagnosticDescriptor CreateDescriptorWithTitle(LocalizableString title
             return new DiagnosticDescriptor(
                 id, title, message,
                 DiagnosticCategory.Style,
-                severity,
+                DiagnosticSeverity.Hidden,
```

Comment:
Why? Is this always true?

Reply:
There is no longer a need to create descriptors with each effective severity, so I just create one with the default and use it in all the places.